docs/platform_introduction/services.md

```diff
@@ -11,4 +11,20 @@ The pipeline can be built up as a directed acyclic graph (DAG) where the lines b
 All these Services can be already provided within a Study Template or can be set up from scratch by the user by selecting options in the Service catalog.
 
 ![catalog](https://user-images.githubusercontent.com/28002886/137334462-d9b5aacf-b329-447a-bd29-110d6d877b9b.png)
-*Connected Services as well as selection of Services via the Service Catalog in the workbench.*
\ No newline at end of file
+*Connected Services as well as selection of Services via the Service Catalog in the workbench.*
+
+
+## Service Types
+Services come in generally two flavors: computational or interactive. Computational Services such as a [**Python Runner**](docs/tutorials/python_runner.md) will allow setting of some parameters (e.g. input files) and will perform a computation with no user interaction required. Interactive Services only perform tasks with user interaction, as is the case for Services such as the [**JupyterLab**](docs/study_setup/JupyterLabs.md). JupyterLab Services will allow you to write and execute code within the JupyterLab GUI, but as with all interactive Services, will not run automatically when a Study pipeline is [**launched**](docs/study_setup/run_simulation.md). Computational Services, on the other hand, will automatically read all input parameters at runtime and execute the static code embedded inside them. 
+
+In terms of viewing, Interactive services will have a special view in the Workbench through the *Interactive* View tab ![interactive](https://user-images.githubusercontent.com/28002886/153265444-6638c003-d90c-4a34-b751-cb0193879993.png ':size=25'). You may occasionally want to maximize this view in order to have a larger working area. To do this, you can click on the ```maximize``` button on the upper right hand corner of the *Interactive View*.
+
+![minimize](https://user-images.githubusercontent.com/28002886/153834101-ed179672-6c23-440f-ac0b-971426f16e39.png ':size=300') 
+
+To regain access the other **Workbench** options such as inspecting Service inputs and outputs or changing the Pipeline, you can minimize the *Interactive View* by clicking on the minimize button on the upper right hand corner.
```

Comment:
MINOR: why you use sometimes capitalize words in the middle of a sentence  e.g. ``changing the *P*ipeline``  and moreover sometimes is bold and sometimes not.
Look in this sentence the words Study/ies, tab and service/s: `` Study in your **Studies** tab with the single Service inside the Study. ... o go through the **Services** Tab.``

Reply:
usually I do this to indicated "protected" osparc words. It's true though that I'm not always consistent... will double check later. Do you prefer non-bold and non capitalized always?

Reply:
no preference :-) thx